client/src/components/home/index.jsx

```diff
@@ -429,12 +429,18 @@ const Home = ({ swaps, updateSwaps, clients, swapPairs, update, setupEth, setupT
                 <CardContent>
                   <form>
                     <div className={classes.tokenContainer + " Element"}>
-                      <div className={classes.balContainer}>
-                        <Typography color="textSecondary" variant="subtitle2">From</Typography>
-                        {bal &&
-                          <Typography color="textSecondary" variant="subtitle2">wallet balance:{" "} {(bal >= 1) ? Math.floor(bal) : bal}{" "}{inputToken.title} </Typography>
-                        }
-                      </div>
+                      <Grid container className={classes.balContainer}>
+                        <Grid item xs={2} md={3} lg={4}>
+                          <Typography color="textSecondary" variant="subtitle2">From</Typography>
+                        </Grid>
+                        <Grid item xs={10} md={9} lg={8}>
+                          {bal &&
```

Comment:
this is a string comparison, it will return true-ish unless `bal` is undefined, please confirm that that's intended vs comparing to a numeric 0.